libs/webgui/pilight.css

```diff
@@ -129,11 +129,26 @@ div.ui-slider-switch, div.screen {
 	right: 10px;
 }
 
+@keyframes blink {
```

Comment:
This requires CSS3, did you check browser support for it?

Reply:
Yes, I tested successfully with:
- Chrome
- Firefox
- IE
- Edge